### ISSUE TITLE
[BUG] cookies flag (either -b or --cookies) does not work properly when there are multiple cookies

### ISSUE THREAD

**Issue by JulianGR** [epi052/feroxbuster#1150]
**Describe the bug**
Both `-b` and `--cookies` don't work properly - cookies don't get sent in requests when there are multiple cookies like so:

```
Cookie: ASP.NET_SessionId=t0asdfv; cache=es; version=2.9.3.0&release=True&nivel=1
```

**To Reproduce**
1. use either `-b` or `--cookies` flag and use multiple cookies

See this screenshot:
![Cookies flag being used, but no cookies](https://github.com/epi052/feroxbuster/assets/53094530/ce2c73e9-b3b5-41c6-8c90-ec1ff7f95f02)

Also, Burp suite with request shows empty cookie header
![3](https://github.com/epi052/feroxbuster/assets/53094530/bf82b8e7-5264-497f-91ff-0ed53ee138e6)

**Expected behavior**
I expect cookies to **actually** be sent

**Traceback / Error Output**
N/A

**Environment (please complete the following information):**
 - feroxbuster version: 2.10.3 (latest), happening since a while ago, so it not new from this latest version.
 - OS: happening in WSL2 Kali, in normal Kali, and Windows. Using Powershell, CMD or bash yields the same error. I tried different terminals to see if quotes or double quotes are being wrongfully parsed, but it does not seem the case.

**Additional context**
N/A

**Comment by JulianGR** [comment]
Yes, redacted for privacy

```
feroxbuster -u https://asdf.com -b 'ASP.NET_SessionId=t0iasdfv; idbox.cache=es; idbox.version=version=2.9.3.0&release=True&nivel=1' --proxy 127.0.0.1
```

**Comment by epi052** [comment]
sorry for the delay, will look into this tonight/tomorrow morning

**Comment by epi052** [comment]
ok, there are a few problems here. 

first, `-b` expects a single key/value pair separated by an equals sign. 

so, a better invocation would be 

```
-b 'ASP.NET_SessionId=t0iasdfv' -b 'idbox.cache=es' ...
```

however, this approach breaks down on your third cookie due to mulitple equals signs. 

thankfully, since `-b` / `--cookies` has always just been sugar around adding `Cookie: ` and some processing logic, you can just use `-H|--headers` directly, which doesn't make any assumptions about the value passed (it does assume there's a `:`, but that's all). 

```
-H 'Cookie: ASP.NET_SessionId=t0iasdfv; idbox.cache=es; idbox.version=version=2.9.3.0&release=True&nivel=1'
```

**Comment by JulianGR** [comment]
Wouldn't it be better that the `-b` flag accepts multiple cookies, with equals sign included? so that it allows faster copy-pasting. Or at least include it in the docs.

**Comment by epi052** [comment]
> Wouldn't it be better that the `-b` flag accepts multiple cookies, with equals sign included? so that it allows faster copy-pasting. Or at least include it in the docs.

yes, i started working on that already :grin: itll be in the next release

**Comment by epi052** [comment]
update:

the following cli options
```
--cookies 'ASP.NET_SessionId=t0iasdfv; idbox.cache=es; idbox.version=version=2.9.3.0&release=True&nivel=1' -b 'stuff=things' -b 'mostuff=mothings;'
```

produce the following header
```
 🤯  Header                │ Cookie: ASP.NET_SessionId=t0iasdfv; idbox.cache=es; idbox.version=version2.9.3.0&releaseTrue&nivel1; stuff=things; mostuff=mothings
```
